modules/cassandra/src/main/java/org/testcontainers/containers/delegate/CassandraDatabaseDelegate.java

```diff
@@ -18,15 +19,19 @@
  */
 @Slf4j
 @RequiredArgsConstructor
+@AllArgsConstructor
 public class CassandraDatabaseDelegate extends AbstractDatabaseDelegate<Session> {
 
     private final ContainerState container;
+    private boolean authenticationEnabled=false;
```

Comment:
missing spaces around `=`

Reply:
Refactored code to remove any changes in CassandraDatabaseDelegate.